chrome/content/zotero/xpcom/style.js

```diff
@@ -81,9 +81,41 @@ Zotero.Styles = new function() {
 		
 		// hidden dir
 		dir.append("hidden");
-		if(dir.exists()) i += _readStylesFromDirectory(dir, true);
+		if (dir.exists()) i += _readStylesFromDirectory(dir, true);
 		
 		Zotero.debug("Cached "+i+" styles in "+((new Date()).getTime() - start)+" ms");
+		
+		// transfer and reset "export.bibliographyLocale" pref value
+		var bibliographyLocale = '';
+		bibliographyLocale = Zotero.Prefs.get("export.bibliographyLocale");
+		if (bibliographyLocale) {
+			Zotero.Prefs.set("export.lastLocale", bibliographyLocale);
+			Zotero.Prefs.set("export.quickCopy.locale", bibliographyLocale);
```

Comment:
Well, only issue with not creating defaults is that `Zotero.Prefs.get` throws on missing preferences. Right now we catch that in various places, but we should probably just change it to return null in that case, maybe with a debug line. There's no point in filling up about:config with empty entries.